src/rdf4cpp/Timezone.hpp

```diff
@@ -63,11 +63,25 @@ struct Timezone {
         return tz;
     }
 
+    // sign, hours, :, minutes
+    // assumes offset stays inside spec (+-14 hours)
```

Comment:
Same thing here, could this be larger?

Reply:
General question: Does chrono stuff not enforce the limits of the datatypes? This would then mean that all buffer calculations would actually need to be based on the underlying datatypes and not on their logical size, right?